### DISCUSSION TITLE
Does radius_graph function removes not connected points?

### DISCUSSION THREAD

**Discussion by KJeziorek** [pyg-team/pytorch_geometric#7552] [Q&A]
As the title suggests, I am working with the radius_graph function on a large dataset consisting of 25,000 points per graph, with a total of 8,000 graphs. When I use the function with non-normalized data.pos, I obtain an average number of edges of 756,736 and 24,457 points. However, when I normalize data.pos by multiplying one of the coordinates by value >1, the average number of edges decreases to 325,592. This decrease makes sense because the points are further apart, resulting in fewer connections. But, the number of points also decreases to 15,007.

Does this mean that the function removes disconnected points? I attempted to find information about this, but unfortunately, I did not succeed.

**Comment by wsad1** [comment]
Why do you say the number of points decreases?  The [RadiusGraph](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.transforms.RadiusGraph.html#torch_geometric.transforms.RadiusGraph) only changes `data.edge_index`, so the number of nodes really shouldn't be affected.
